Xlib/ext/composite.py

```diff
@@ -206,6 +206,29 @@ def name_window_pixmap(self):
     cls = self.display.get_resource_class('pixmap', drawable.Pixmap)
     return cls(self.display, pid, owner = 1)
 
+class CompositeGetOverlayWindow(rq.ReplyRequest):
```

Comment:
Class `CompositeGetOverlayWindow` could be just `GetOverlayWindow` for consistency. The rest looks fine to me.